code/modules/mob/living/carbon/xenomorph/update_icons.dm

```diff
@@ -138,14 +139,47 @@
 
 	wound_overlay.vis_flags &= ~VIS_HIDE // Show the overlay
 
+/mob/living/carbon/xenomorph/proc/update_snowflake_overlays()
+	if(!backpack_overlay)
+		return
+	if(!back)
+		backpack_overlay.icon_state = ""
+		return
+	if(!istype(back,/obj/item/storage/backpack/marine/duffelbag/xenosaddle))
+		backpack_overlay.icon_state = ""
+		return
+	var/obj/item/storage/backpack/marine/duffelbag/xenosaddle/saddle = back
+	if(stat == DEAD)
+		backpack_overlay.icon_state = "[saddle.style][(xeno_flags & XENO_ROUNY) ? " rouny" : ""] Dead"
+	else if(lying_angle)
+		if((resting || IsSleeping()) && (!IsParalyzed() && !IsUnconscious() && health > 0))
+			backpack_overlay.icon_state = "[saddle.style][(xeno_flags & XENO_ROUNY) ? " rouny" : ""] Sleeping"
+		else
+			backpack_overlay.icon_state = "[saddle.style][(xeno_flags & XENO_ROUNY) ? " rouny" : ""] Knocked Down"
+	else
+		backpack_overlay.icon_state = "[saddle.style][(xeno_flags & XENO_ROUNY) ? " rouny" : ""]"
+
 /mob/living/carbon/xenomorph/update_transform()
 	..()
 	return update_icons()
 
-///Used to display the xeno wounds without rapidly switching overlays
+///Used to display xeno wounds & equipment without rapidly switching overlays
+
+/atom/movable/vis_obj/xeno_wounds/backpack_overlay
+	layer = ABOVE_MOB_LAYER
+	///The xeno this overlay belongs to
+	var/mob/living/carbon/xenomorph/owner
```

Comment:
For future reference please have your vars beneath every single non variable (ie. `var/whatever = whatever` goes below `icon = blabla`)